packages/amplify-e2e-tests/src/__tests__/init-special-case.test.ts

```diff
@@ -72,4 +74,12 @@ describe('amplify init', () => {
       getParameters(projectRoot, category, authResourceName);
     }).not.toThrow();
   });
+
+  it('should fail if running amplify init -y on already initialized project', async () => {
```

Comment:
Please work with @bombguy and @0618 to get this included in codebuild manifests and tested in e2e test run.

Reply:
Well. Skip the codebuild manifest part, since it's addition to already existing test file. But please obtain e2e run where this test passes.